ui/plugins/controller.js

```diff
@@ -439,6 +439,7 @@ treeherder.controller('PluginCtrl', [
 
         var selectJobAndRender = function(job) {
             $scope.jobLoadedPromise = selectJob(job);
+            $( "#info-panel" ).addClass( "info-panel-slide" );
```

Comment:
I don't think we want the extra spacing inside parens; we aren't doing that anywhere else